### ISSUE TITLE
Have two DateTime times, NaiveDateTime and DateTime?

### ISSUE THREAD

**Issue by AutumnalDream** [tartiflette/tartiflette#323]
## Request a feature

* A part of the project could be improved?

Would it make sense to have two DateTime scalars, one timezone naive and one not?

**Comment by zer0tonin** [comment]
@AutumnalDream it might be better suited for the plugin https://github.com/tartiflette/tartiflette-plugin-scalars

**Comment by Maximilien-R** [comment]
I agree with @zer0tonin. Especially since we are thinking about removing the `Date` and `DateTime` scalars provided by `tartiflette` since both of them are not part of the [GraphQL specification](https://graphql.github.io/graphql-spec/June2018/#sec-Scalars). Thus, it makes more sense to distribute them through plugins such as [`tartiflette-plugin-scalars`](https://github.com/tartiflette/tartiflette-plugin-scalars).
